Automattic/jetpack
# Bypass custom link check for galleries created in classic block

#### Changes proposed in this Pull Request:

* Checks for existence of `.attr( 'data-permalink' )` before running check for custom image links in carousel. The recent modification to allow for custom links to bypass the carousel in the gallery block refactor broke the carousel for galleries created with the classic editor/block.

Fixes: #18694

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:

- Apply this PR to local install
- Add a gallery via the classic editor or classic block and check that images open in lightbox
- Also add a gallery block and check the same
-  Run Gallery refactor PR locally as well as this PR and check that images with custom links forward to the link instead of opening in lightbox, but other images still open in lightbox

#### Proposed changelog entry for your changes:
* No changelog entry needed
